tests/test_convert.py

```diff
@@ -174,6 +177,37 @@ def test_revealjs_offline_converter(
         ]:
             assert (assets_dir / file).exists()
 
+    def test_revealjs_offline_inlining(
+        self,
+        tmp_path: Path,
+        presentation_config: PresentationConfig,
+        mocker: MockerFixture,
+    ) -> None:
+        # Mock requests.Session.get to return a fake response
+        mock_response = mocker.Mock()
+        mock_response.text = "body { background-color: #9a3241; }"
+        mock_response.content = b"body { background-color: #9a3241; }"
+        mocker.patch.object(requests.Session, "get", return_value=mock_response)
+
+        out_file = tmp_path / "slides.html"
+        RevealJS(presentation_configs=[presentation_config], offline="true").convert_to(
+            out_file
+        )
+        assert out_file.exists()
+
+        with open(out_file, encoding="utf-8") as file:
+            content = file.read()
+
+        soup = BeautifulSoup(content, "html.parser")
+
+        # Check if CSS is inlined
+        styles = soup.find_all("style")
+        assert any("background-color: #9a3241;" in style.string for style in styles)
+
+        # Check if JS is inlined
+        scripts = soup.find_all("script")
+        assert any("background-color: #9a3241;" in script.string for script in scripts)
+
```

Comment:
Like for other tests, I would also add a logic to assert that **not** setting the 
*one file* flag will correctly download and save assets in other files.

Reply:
This is already checked by `test_revealjs_offline_converter`, right?

Reply:
Yes, sort of, but I prefer having another tests that checks this, even if the tests overlap. We never test too much :-)

Reply:
You are right, however I don't quite understand what test you want me to add. Do I copy and paste the `test_revealjs_converter` and `test_revealjs_offline_converter` tests again? I am a bit confused.

Reply:
You can extend the current test, with something like

```python
RevealJS(
            presentation_configs=[presentation_config], offline="false", one_file="true"
        ).convert_to(out_file)
        assert out_file.exists()
```
and check that the output file contains data-uri encoded videos, but not inlined CSS / JS.

Reply:
Done!